.ci.yaml

```diff
@@ -5068,6 +5068,10 @@ targets:
       subshard: libraries
       tags: >
         ["framework", "hostonly", "shard", "windows"]
+      env_variables: >-
+        {
+          "NO_OP": "true"
```

Comment:
Is this same as `LEAK_TRACKING`? If yes, I am not sure if we want this temp variable to validate considering these targets are in staging already. Namely use `LEAK_TRACKING` directly.

Reply:
Nope. "NO_OP" is not consumed. Just arbitrary name.
I want to make sure the mechanism of passing the variable will not break jobs.
If it will not, I will pass LEAK_TRACKER.

I am very conservative :)

Reply:
It should not. It's just an env. variable

 The logic to parse env. variables have landed weeks ago, and Packages repo have been running without any issue: https://github.com/flutter/packages/blob/main/.ci.yaml (see the env_varaibles).

But up to you if you want to go with this first.

Reply:
OK, switched to LEAK_TRACKING.